python/TestHarness/RunParallel.py

```diff
@@ -148,6 +149,16 @@ def startReadyJobs(self, slot_check):
       os.chdir(saved_dir)
       sys.path.pop()
 
+  def getOutputFromFiles(self, tester):
+    # TODO: Use redirect_stdout when ready (libMesh update required Ref: https://github.com/libMesh/libmesh/issues/1159)
+    # file_list = glob.glob(os.path.join(tester.specs['test_dir'], tester.specs['redirect_stdout'] + '.*'))
+    file_list = glob.glob(os.path.join(tester.specs['test_dir'], 'stdout.processor.*'))
```

Comment:
So you're saying instead of using a wildcard, ask the TestHarness how many processors we are currently allocating to the job and generate a list based on those results. I can't think of why that would not work.

If that sounds correct, I'll start down that direction.